components/molecules/MarkdownWrapper/markdown-wrapper.tsx

```diff
@@ -0,0 +1,39 @@
+import ReactMarkdown from "react-markdown";
+import SyntaxHighlighter from "react-syntax-highlighter";
+import { atomOneDark } from "react-syntax-highlighter/dist/cjs/styles/hljs";
+
+interface MarkdownWrapperProps {
+  content: string;
+  pClassName?: string;
+}
+
+const MarkdownWrapper = ({ content, pClassName }: MarkdownWrapperProps) => (
```

Comment:
For this could you please create a storybook ??

Reply:
I committed a storybook for this component. It is my first time using storybook so please let me know if there is any issues with it. Thank you.